src/tests/encrypt-no-change/out/.keboola/manifest.json

```diff
@@ -10,6 +10,9 @@
     "config": "{component_type}/{component_id}/{config_name}",
     "configRow": "rows/{config_row_name}"
   },
+  "allowedBranches": [
+    "*"
+  ],
```

Comment:
tak to `allowedBranches` pridam aj do `encrypt` testu ktory vznikol v https://github.com/keboola/keboola-as-code/pull/82